apollo-configservice/src/test/java/com/ctrip/framework/apollo/configservice/service/ReleaseMessageServiceWithCacheTest.java

```diff
@@ -156,8 +157,8 @@ public void testWhenReleaseMsgSizeBiggerThan500() throws Exception {
             .findLatestReleaseMessagesGroupByMessages(Sets.newHashSet(someMsgContent, antherMsgContent));
```

Comment:
Good point, how about we change the input to `LinkedHashSet`? That might look more elegant.

Reply:
Thanks for reply.
Yes it will also works. It just depends on whether you want the assertion to be depend on order or not. I have modified the fix to use `LinkedHashSet`.